tests/nested/core/connected-after-reboot-revert/task.yaml

```diff
@@ -46,13 +45,28 @@ execute: |
   remote.exec "sudo snap install --channel=\"$BLUEZ_CHANNEL\" bluez"
   remote.exec "snap connections bluez" | MATCH 'uhid  *bluez:uhid  *:uhid'
 
-  chg_id=$(remote.exec 'sudo snap install --dangerous --transaction=all-snaps --no-wait ./pc-kernel_badhook.snap /var/lib/snapd/snaps/snapd_*.snap')
+  if [ "$VERSION" -eq 16 ]; then
```

Comment:
Thanks, I didn't know that, but note that in some cases I'm checking if the version is bigger than something (`"$VERSION" -ge 20`). Is there a way to do that with some auxiliary command? Otherwise I'm not sure if it is worth it to change to use `tests.nested is-nested` in some places and checking against $VERSION in others. Also, I'm using $VERSION to get the channel for some snaps.

Reply:
well, if you already got the $VERSION for the channel I think that the -eq check in ok. 
This is a good idea to have a command to validate -ge so we can check for core 20 an 22 in a single check.